Makefile

```diff
@@ -1,10 +1,14 @@
 BASE_BRANCH ?= devel
+PROJECT := shipyard
```

Comment:
This could be calculated as `$(notdir $(CURDIR))`, and declared in `Makefile.dapper` instead, so we don’t need to manually specify it everywhere.

Reply:
Thanks :)

I had a feeling there's some automatic way to do this and that you'll suggest it ;)